components/LeagueCard/LeagueCard.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { cn } from '@/utils/utils';
+import Image from 'next/image';
+import placeholderImage from './placeholderImage.svg';
+
+interface ILeagueCardProps extends React.HTMLAttributes<HTMLDivElement> {
+  isEliminated?: boolean;
+}
+
+const LeagueCard = React.forwardRef<HTMLDivElement, ILeagueCardProps>(
+  ({ className, isEliminated = false, ...props }, ref) => (
```

Comment:
I'm not sure why all of these components need a custom `className`. It's always going to have the same static style.

Reply:
Also, give each one of these components a unique className. I dislike looking at the DOM and only seeing Tailwind classes. It doesn't allow for ease of debugging when we find bugs. Set the className to the component name.